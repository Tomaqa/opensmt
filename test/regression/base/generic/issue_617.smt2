(set-option :produce-models true)
(set-logic QF_LIA)
(declare-const x Int)
(define-fun f () Bool (= x 1))
(assert (= x 1))
(check-sat)
(get-value (f))
(get-value ((+ (* 2 x) 1)))
(get-value ((+ 1 1)))
(get-value ((!(+ x 1) :named test)))
(get-value ((let ((y 0)) y)))
(get-value ((let ((y 1) (z 2)) (+ y z))))
