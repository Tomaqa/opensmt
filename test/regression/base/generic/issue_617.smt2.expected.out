sat
((f true))
(((+ (* 2 x) 1) 3))
(((+ 1 1) 2))
(((!(+ x 1) :named test) 2))
(((let ((y 0)) y) 0))
(((let ((y 1) (z 2)) (+ y z)) 3))
